app/models/zone.rb

```diff
@@ -36,6 +38,11 @@ class Zone < ApplicationRecord
   include AggregationMixin
   include ConfigurationManagementMixin
 
+  scope :visible, -> { where(:visible => true) }
+  default_value_for :visible, true
+
+  MAINTENANCE_ZONE_NAME_PREFIX = '__maintenance__'.freeze
```

Comment:
Does this string need to be a constant since it's only used in one place on create?

Reply:
yes, every constant is better than "automagic" value in code. 
Moreover it's used in `zone_spec.rb`

Reply:
No one else will be using this string, right?

Reply:
Whether it is a "magic value" assigned to a constant and used in a single method versus a local variable in a method I don't see the benefit of the constant TBH.

We should be compartmentalizing the logic, no one external to the create method should really care about the prefix used to generate the unique names.  Making sure no one in a year comes starts using the prefix in an incorrect way because it is available outweighs having a local string variable in a method.